build-tests/api-documenter-test/etc/markdown/api-documenter-test.docbaseclass.md

```diff
@@ -6,6 +6,7 @@
 
 Example base class
 
+
 <b>Signature:</b>
```

Comment:
Is adding this blank line intentional? Seems like it shouldn't be.

Reply:
Removed this blank line although it was autogenerated when I ran the test. So with future runs it might get added again.

Reply:
That makes it seem like a regression was introduced.

Reply:
That would be highly unlikely because we did not touch the `MarkdownDocumenter` in this PR. I added a `docCategory` tag to the test files that might have somehow influenced the output.